spec/forms/candidate_interface/immigration_status_form_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::ImmigrationStatusForm, type: :model do
+  describe '#validations' do
```

Comment:
seems like this is all wrapped in this block and we need a test or two for validations